src/backends/zdk/transformUtils/transformV3AskEvent.ts

```diff
@@ -0,0 +1,56 @@
+import { V3AskEvent, V3AskEventType } from '@zoralabs/zdk/dist/queries/queries-sdk';
+import {
+  FIXED_PRICE_EVENT_TYPES,
+  FIXED_PRICE_MARKET_SOURCES,
+  FIXED_SIDE_TYPES,
+  MarketFixedPriceEvent,
+} from '../../../types';
+import { TransformFunctionParam } from './transformMarketEvent';
+
+export function transformV3AskEvent({
+  event,
+  at,
+  blockInfo,
+  price,
+  eventType,
+}: TransformFunctionParam<V3AskEvent, MarketFixedPriceEvent['eventType']>):
+  | MarketFixedPriceEvent
+  | undefined {
+  let fixedPriceEvent: FIXED_PRICE_EVENT_TYPES | undefined = undefined;
```

Comment:
could be cleaner here to break this out to a small function with if return vs a switch. this looks good tho